api-admin/src/main/java/bio/terra/pearl/api/admin/service/portal/PortalExtService.java

```diff
@@ -79,4 +84,9 @@ public PortalEnvironment updateEnvironment(
     portalEnv.setSiteContentId(updatedEnv.getSiteContentId());
     return portalEnvironmentService.update(portalEnv);
   }
+
+  public void removeUserFromPortal(UUID userId, String portalShortcode, AdminUser operator) {
```

Comment:
Here and in other places I'd lean towards using `adminUserId` as the variable name for clarity
```suggestion
  public void removeUserFromPortal(UUID adminUserId, String portalShortcode, AdminUser operator) {
```

Reply:
Yup, sorry.. I was trying to be consistent with the method names, the consistent among my changes. :-) Will change.